Changelogs for Digital Twin Manager Plugins 1.3.1 (#2693)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020-1-20-the-configured-properties-are-not-visible-on-the-edit-widget-screen.md

---
date: ""

> title: "The configured properties are not visible on the Edit Widget screen."

Reviewer: ```suggestion
title: "Configured properties are now visible when editing the Asset properties widget"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1666"
version: "1020.1.20"
---

> Previously, configured asset properties were not visible when editing the 'Asset Properties Widget', requiring users to re-add them. Now, users can see existing configured properties on edit and add new properties as needed.

Reviewer: ```suggestion
Previously, configured asset properties were not visible when editing the "Asset properties" widget, requiring users to re-add them. Now, users see existing configured properties on editing and can add new properties as needed.
```
